storage/innobase/xtrabackup/src/xbcrypt_common.c

```diff
@@ -207,6 +207,17 @@ xb_crypt_cipher_open(gcry_cipher_hd_t *cipher_handle)
 	return 0;
 }
 
+/************************************************************************
+Mask the argument value. This is to avoid showing secret data on command
+line output */
```

Comment:
```suggestion
/* Mask the argument value. This is to avoid showing secret data on command
line output
@param[in, out]         argument    argument to be masked
@param[in,out]          opt               original argument which is reallocated
*/
```